packages/@lightningjs/ui-components/src/components/TabBar/TabBar.test.js

```diff
@@ -354,4 +354,16 @@ describe('TabBar', () => {
       expect(tabBar._Tabs.selectedIndex).toBe(1);
     });
   });
+
+  it('should optionally allow the selected tab to remain selected when tab bar loses focus', () => {
+    expect(tabBar._Tabs.items[0].mode).toBe('focused');
+    expect(tabBar._Tabs.items[1].mode).toBe('unfocused');
+
+    tabBar.retainSelection = true;
+    tabBar.mode = 'unfocused';
+    tabBar._updateTabs();
```

Comment:
in general, we try to avoid directly calling `_update<Name>` methods directly as this doesn't simulate the component update cycle - instead we use `testRenderer.forceAllUpdates()`. You can see it used in the tests here for example https://github.com/rdkcentral/Lightning-UI-Components/blob/develop/packages/%40lightningjs/ui-components/src/components/Button/Button.test.js